### PR TITLE
Time parser was confusing months and minutes

### PR DESCRIPTION
**Pull request by victorhtc**

### CODE REVIEW

File: components/messages/Discussion.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default class Discussion extends Component {
                 avatarUrl: "/static/avatar.jpg",
                 userid: null,
                 userName: "Fred",
-                date: new Date()
+                date: '2017-10-09T16:38:05.680Z'
```
Comment (dcorb) [inline components/messages/Discussion.js]: Probably we should do the same on commenData (some lines above)

File: components/date/DateUtils.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function formatDate(date, dateFormat) {
 }
 
 export function getTime(date) {
-    let theTime = format(date, "h:MM A")
+    let theTime = format(date, "h:mm A")
```
Comment (dcorb) [inline components/date/DateUtils.js]: I like hour:month format, that's how we do it in Europe, don't know about Goiânia

File: components/messages/Discussion.js (hunk 1 above)
Comment (victorhtc) [reply components/messages/Discussion.js]: This is just an example of how it should be flexible. The `date` prop should accept Date or ISO 8601 string. We should keep the previous example as it is.

File: components/date/DateUtils.js (hunk 2 above)
Comment (victorhtc) [reply components/date/DateUtils.js]: lol

**Pull request merged**